modules/tinymce/src/themes/silver/test/ts/webdriver/editor/DisabledMenuItemTest.ts

```diff
@@ -0,0 +1,55 @@
+import { Mouse, RealKeys, UiFinder, Waiter } from '@ephox/agar';
+import { before, describe, it } from '@ephox/bedrock-client';
+import { McEditor, TinyUiActions } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+
+import TablePlugin from 'tinymce/plugins/table/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('webdriver.tinymce.themes.silver.editor.menubar.DisabledMenuItemTest', () => {
+
+  before(() => {
+    Theme();
+    TablePlugin();
+  });
+
+  const settings = {
+    base_url: '/project/tinymce/js/tinymce',
+    plugins: 'table',
+    menu: {
+      table: { title: 'Table', items: 'inserttable | cell row column | tableclass tablecellclass tablecellvalign tablecellborderwidth tablecellborderstyle tablecaption tablecellbackgroundcolor tablecellbordercolor | advtablesort | tableprops deletetable' }
+    },
+    menubar: 'table',
+  };
+
+  const pOpenTableMenu = (editor) => {
+    const tableMenuItemSelector = '[role="menuitem"]:contains("Table")';
+    TinyUiActions.clickOnMenu(editor, tableMenuItemSelector);
+    return Waiter.pTryUntil('Wait for a specific menu to open', () => UiFinder.exists(SugarBody.body(), tableMenuItemSelector));
+  };
+
+  const assertVerticalAlignMenuIsNotOpen = () => {
+    UiFinder.notExists(SugarBody.body(), '[role="menuitemcheckbox"]:contains("None")');
+    UiFinder.notExists(SugarBody.body(), '[role="menuitemcheckbox"]:contains("Top")');
+    UiFinder.notExists(SugarBody.body(), '[role="menuitemcheckbox"]:contains("Middle")');
+    UiFinder.notExists(SugarBody.body(), '[role="menuitemcheckbox"]:contains("Bottom")');
+  };
+
+  it('TINY-7700: Disabled menu item with children should not open on mouse hover', async () => {
+    const editor = await McEditor.pFromSettings(settings);
```

Comment:
Should this be done with `TinyHooks`? It looks like you're using the same editor config for all the tests

Reply:
My first go was using `TinyHooks`. I changed it cause for some reason (dunno) it was breaking `pSendKeysOn` from the last test. It was throwing an error about the selector `'[title="Vertical align"]'`. The only thing I can think of is that the previous editor was causing that. So I used the `McEditor` to add/remove the editor in each test and it worked. Dunno if that makes sense 🤷

Reply:
We want to avoid instantiating editors whenever we can, so using `McEditor` sparingly. Often when using `TinyHooks` we need to tidy up between each test (closing dialogs, menus, clearing editor content). This can sometimes be done in the `beforeEach` and `afterEach` hooks.

See my other comments as to why your test wasn't working